log/loggers.go

```diff
@@ -0,0 +1,193 @@
+package log
+
+import (
+	"errors"
+	"fmt"
+	"log/syslog"
+	"os"
+
+	"github.com/daviddengcn/go-colortext"
+)
+
+// LoggerInfo is a structure that should be used when creating a logger.
+// It contains parameters about how to log (with time, colors, ...) and
+// embeds the Logger interface, which should define how the logger should log.
+type LoggerInfo struct {
+	// These are information-debugging levels that can be turned on or off.
+	// Every logging greater than 'debugLvl' will be discarded . So you can
+	// Log at different levels and easily turn on or off the amount of logging
+	// generated by adjusting the 'debugLvl' variable.
+	debugLvl int
```

Comment:
It is hard to see how a caller outside of onet would subscribe to "only messages less than or equal to 3", since debugLvl is not exported. Perhaps I am missing something about how this is designed to be used?